src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,29 @@
+package duke.command;
+
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+
+/**
+ * Stores the index of the task to be deleted from TaskList.
+ */
+public class DeleteCommand extends Command {
+    private int indexToDelete;
+
+    /**
+     * @param number 1 based indexing for the task to be deleted.
+     */
+    public DeleteCommand(int number) {
+        this.indexToDelete = number - 1;
```

Comment:
I like how you add methods even for such simple arithmetic operations as it makes your code more understandable.